fuzz/fuzz_helper/src/lib.rs

```diff
@@ -174,6 +174,8 @@ pub fn process_ctap_structured(data: &[u8], input_type: InputType) -> arbitrary:
     let mut env = TestEnv::new();
     let mut state = CtapState::new(&mut env, CtapInstant::new(0));
 
+    env.rng().seed_rng_from_u64(u64::arbitrary(unstructured)?);
```

Comment:
Actually, we should actually move this right after the environment creation. The reason being that if CtapState uses `env.rng` we want it to use the seed from the fuzz input already.

The usual pattern for testing should always be:
```rust
// Create the test environment.
let mut env = TestEnv::new();
// Customize the test environment.
env.foo().set_bar();
// Use the test enviroment.
quux(&mut env);
// Possibly inject errors.
env.foo().next_bar(...);
// Continue using the test environment.
quux(&mut env);
```

Reply:
done